app/controllers/concerns/client_sortable.rb

```diff
@@ -14,10 +14,13 @@ def filtered_and_sorted_clients(default_order: nil)
     clients = clients.where(tax_returns: { assigned_user: limited_user_ids }) unless limited_user_ids.empty?
     clients = clients.where(tax_returns: { year: @filters[:year] }) if @filters[:year].present?
     clients = clients.where(tax_returns: { status: @filters[:status] }) if @filters[:status].present?
+    clients = clients.where("intakes.locale = :language OR intakes.preferred_interview_language = :language", language: @filters[:language]) if @filters[:language].present?
+    clients = clients.where(tax_returns: { service_type: "online_intake" }) if @filters[:exclude_dropoffs].present?
     clients = clients.where(intake: Intake.where(had_unemployment_income: "yes")) if @filters[:unemployment_income].present?
+
     if @filters[:vita_partner_id].present?
       id = @filters[:vita_partner_id].to_i
-      clients = clients.where("vita_partners.id = ? OR vita_partners.parent_organization_id = ?", id, id)
+      clients = clients.where('vita_partners.id = :id OR vita_partners.parent_organization_id = :id', id: id)
```

Comment:
What is this `where` change suppose to accomplish? Was the previous syntax with the three arguments incorrect?

Reply:
No, it wasn't broken, but I've since learned about a modified updated rails/active record syntax that we're leveraging in another clause. This is now using "named" parameters using symbols for the id instead of using the ? replacement. It's slightly more "Railsy", and continues to pass the tests.